pkg/pipelines/config/config_test.go

```diff
@@ -76,79 +84,6 @@ func TestManifestWalk(t *testing.T) {
 	}
 }
 
-func TestManifestWalkCallsCICDEnvironmentLast(t *testing.T) {
```

Comment:
Will this test be replaced with a new one?

Reply:
This wont be necessary as the the cicd environment is now present in a separate  struct, so the sorting is not necessary.